### DISCUSSION TITLE
FullPageOs Pi5

### DISCUSSION THREAD

**Discussion by kuzgun61** [guysoft/FullPageOS#528] [Q&A]
Hello, there are 3 different versions of FullpageOs in the PiOs imaging application, but I could not activate any of them on Pi5. After installation, the Debian desktop appears and nothing else appears. Will there be updates for Pi 5?

**Comment by guysoft** [comment]
The current stable version does not have support for Rpi 5.
The nightly one does, but it also has a bug, make sure to set the default user to "pi", otherwise it will fail

**Reply by kuzgun61** [reply]
It's a great system for the project I have, thank you very much.
